Purchases/Attribution/RCAttributionTypeFactory.m

```diff
@@ -0,0 +1,55 @@
+//
+// Created by Andrés Boedo on 2/25/21.
+// Copyright (c) 2021 Purchases. All rights reserved.
+//
+
+#import "RCAttributionTypeFactory.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@implementation RCAttributionTypeFactory
+
+- (Class<FakeAdClient> _Nullable)adClientClass {
+    return (Class<FakeAdClient> _Nullable)NSClassFromString(@"ADClient");
```

Comment:
It shouldn't, or at least it hasn't been an issue yet - this line isn't new, it was just moved from https://github.com/RevenueCat/purchases-ios/blob/develop/Purchases/Purchasing/RCAttributionFetcher.m#L109 and has been in production for a while

Reply:
true, sorry for the noise, I wanted to add it on the "ATTrackingManager", but yeah I don't think it will be in an issue since this one "ADClient" is not giving problems